test/functional/address_types.py

```diff
@@ -195,5 +225,40 @@ def run_test(self):
                 to_node %= 4
                 assert_equal(new_balances[to_node], old_balances[to_node] + to_send * 10 * (2 + n))
 
+        # Get one p2sh/segwit address from node2 and two bech32 addresses from node3:
+        to_address_p2sh = self.nodes[2].getnewaddress()
+        to_address_bech32_1 = self.nodes[3].getnewaddress()
+        to_address_bech32_2 = self.nodes[3].getnewaddress()
+
+        # Fund node 4:
+        self.nodes[5].sendtoaddress(self.nodes[4].getnewaddress(), Decimal("1"))
+        self.nodes[5].generate(1)
+        sync_blocks(self.nodes)
+
+        assert_equal(self.nodes[4].getbalance(), 1)
+
+        self.log.debug("Nodes with change_type=bech32 always use a P2WPKH change output:")
+        self.test_change_output_type(2, [to_address_bech32_1], 'bech32')
+        self.test_change_output_type(2, [to_address_p2sh], 'bech32')
+
+        self.log.debug("Nodes with addresstype=bech32 always use a P2WPKH change output:")
+        self.test_change_output_type(3, [to_address_bech32_1], 'bech32')
+        self.test_change_output_type(3, [to_address_p2sh], 'bech32')
```

Comment:
I don't see why this is the case. No output is native segwit?

Reply:
`addresstype` implicitly sets `changetype`:

`g_change_type = ParseOutputType(gArgs.GetArg("-changetype", ""), g_address_type)`

If the user overrides `changetype` it will use that, so I could change the word "always" in the comment to reflect that better.

Reply:
riiiiiiiiiiight. Could you just write that? I actually forgot this and I've reviewed all the PRs so far :(

Reply:
add at end of above comment "unless changetype is set otherwise"

Reply:
Done

Reply:
I think we can discuss what to do with v1 witnesses when the time comes.